website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -95,90 +148,51 @@ Advanced users can configure multiple projects to use the same Cloud CLI executa
 
 </TabItem>
 
 </Tabs>
 
+
 ## Update dbt Cloud CLI
 
 The following instructions explain how to update the dbt CLoud CLI to the latest version depending on your operating system. 
 
-<Tabs>
-	
-<TabItem value="mac" label="macOS">
-
-To update the dbt Cloud CLI, run `brew upgrade dbt-cloud-cli`. 
+During the public preview period, we recommend updating before filing a bug report. This is because the API is subject to breaking changes.
 
-</TabItem>
-<TabItem value="windowslinux" label="Windows and Linux">
 
-To update, follow the same process explained in [Install manually (Windows)](/docs/cloud/cloud-cli-installation?install=windows#install-dbt-cloud-cli) and replace the existing `dbt.exe` executable with the new one.
+<Tabs>
+<TabItem value="existing" label="Existing dbt Core users (pip)">
 
+To update:
+- Make sure you're in your virtual environment
+- Run `pip install --upgrade dbt`.
+	
 </TabItem>
 
-</Tabs>
-
-## Configure the dbt Cloud CLI
-
-After installation, you can configure the dbt Cloud CLI for your dbt Cloud project and use it to run [dbt commands](/reference/dbt-commands) similar to dbt Core. For example, you can execute the following command to compile a project using dbt Cloud:
-
-```bash
-dbt compile
-```
-
-**Prerequisites**
-
-- You must set up a project in dbt Cloud.
-- You must have your [personal development credentials](/docs/dbt-cloud-environments#set-developer-credentials) set for that project. The dbt Cloud CLI will use these credentials, stored securely in dbt Cloud, to communicate with your data platform.
-- You must [enroll](/docs/dbt-versions/experimental-features) in the dbt Cloud beta features. 
-	- To enroll, navigate to your **Profile Settings** and enable the **Beta** flag under **Experimental Features**.
-
-Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt Cloud project.
+<TabItem value="mac" label="macOS (brew)">
 
-1. Ensure you meet the prerequisites above.
-2. Create an environment variable with your [dbt Cloud API key](/docs/dbt-cloud-apis/user-tokens):
-   - On MacOS, Linux, or Windows add an environment variable:
-
-        ```bash
-        export DBT_CLOUD_API_KEY="1234" # Replace 1234 with your API key   
-        ```
-
-   - In Powershell, add an environment variable: 
-     - Note that this variable resets if you restart your shell. To add an environment variable permanently, add a system environment variable in your platform.
-
-3. Navigate to a dbt project in your terminal:
-
-```bash
-cd ~/dbt-projects/jaffle_shop
-```
-
-4. In your `dbt_project.yml` file, ensure there is a section titled `dbt-cloud`. This section is required to have a `project-id` field with a valid project ID. 
-
-```yaml
-# dbt_project.yml
-name:
+To update the dbt Cloud CLI, run `brew upgrade dbt-cloud-cli`. 
```

Comment:
per benoit:

-t his command should be same command as intall one: `brew install dbt` vs `brew upgrade dbt-cloud-cli ` -- is it `dbt` or `dbt-cloud-cli`?
- to update in brew, users can run `brew install` to install latest version. is it `brew upgrade` or `brew install`?

Reply:
either works, but I believe the standard is brew upgrade. it should also be `brew install dbt`, good catch